src/main/java/nextstep/courses/domain/Student.java

```diff
@@ -0,0 +1,59 @@
+package nextstep.courses.domain;
+
+import java.util.Objects;
+
+public class Student {
+    private final String studentId;
+    private SelectStatus selectedStatus = SelectStatus.UNSELECTED;
+
+    public static Student SelectedStudent(String studentId) {
```

Comment:
메서드 이름은 관례에 따라 소문자로 시작하도록 작성해보면 좋을 것 같습니다.
```suggestion
    public static Student selectedStudent(String studentId) {
```